.github/workflows/advice.js

```diff
@@ -62,7 +62,7 @@ gh pr checkout ${issue_number}
   }
 
   const dcoCheck = await getDcoCheck(github, owner, repo, sha);
-  if (dcoCheck?.conclusion !== "success") {
```

Comment:
If `dcoCheck` is `undefined`, `dcoCheck?.conclusion` evaluates to `undefined`, which is not "success".